terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,9 @@
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+
+# service configs
+use_set_environment_files = true
+log_level = "trace"
+include_api_filing_public_specs = "1"
+include_pending_public_specs = "1"
+include_private_specs = "1"
```

Comment:
Can these be removed if they're being supplied by the global config?

Reply:
Yes they can, I'll remove them for `cidev` only for now